doc-surrealdb_versioned_docs/version-1.x/surrealql/parameters.mdx

```diff
@@ -30,103 +62,221 @@ let people = await surreal.query("SELECT * FROM article WHERE status INSIDE $sta
-                Represents values from the session functions as an object
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$before</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value before a mutation on a field
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$after</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value after a mutation on a field
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$value</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the value after a mutation on a field (identical to $after in the case of an event)
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$input</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the initially inputted value in a field definition, as the value clause could have modified the $value variable
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$this</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the current record in a subquery
-            </td>
-        </tr>
-        <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$parent</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the parent record in a subquery
-            </td>
-        </tr>
-         <tr>
-            <td colspan="2" scope="row" data-label="Name">
-                <code>$event</code>
-            </td>
-            <td colspan="2" scope="row" data-label="Description">
-                Represents the type of table event triggered on an event.
-            </td>
-        </tr>
-    </tbody>
-</table>
\ No newline at end of file
+SurrealDB uses predefined variables. For that purpose, you can use those variables inside your query but you cannot declare new parameters using one of the following names:
+
+### $auth
+
+Represents the currently authenticated scope user.
+
+```surql
+DEFINE TABLE user SCHEMAFULL
+	PERMISSIONS
+		FOR select, update, delete WHERE id = $auth.id;
+```
+
+### $token
+
+Represents values held inside the JWT token used for the current session.
+
+```surql
+DEFINE TABLE user SCHEMAFULL
+  PERMISSIONS FOR select, update, delete, create
+  WHERE $scope = "users"
+  AND email = $token.email
+;
```

Comment:
chore: remove line return

Reply:
I copied this sample from somewhere else but true, it is a bit ugly that way. Will take it out